services/ui/src/components/Breadcrumbs/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Link from 'next/link';
+
+export default ({project, environment}) => {
+  const projectURL = project ? `/project/${project.replace(/\s+/g, '-').toLowerCase()}` : ';'
+  const environmentURL = environment ? `/project/${project}/${environment}`.replace(/\s+/g, '-').toLowerCase() : '';
```

Comment:
Query strings are fine for now. Can get rid of these constants since they won't be used.